backend/users.js

```diff
@@ -1,65 +1,66 @@
-import React, { useState, useEffect } from 'react';
-import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
-import { ApolloProvider, InMemoryCache, ApolloClient } from '@apollo/client';
-import Welcome from './Welcome Page/welcome';
-import Login from './Login Page/Login';
-import Signup from './Signup Page/Signup';
-import { UserContext } from './UserContext';
-import Home from './Home Page/Home';
-import ProfileName from './ProfileSetup/ProfileName';
-import ProfileLocation from './ProfileSetup/ProfileLocation';
-import ProfileJobTitle from './ProfileSetup/ProfileJobTitle';
-import Profile from './Profile Page/Profile';
-import ProfileView from './Profile Page/ProfileView';
-import Friends from './Friends Page/Friends';
-import NotificationCenter from './Notifications/NotificationCenter';
+// backend/users.js
 
-const client = new ApolloClient({
-    uri: 'http://localhost:4000/graphql', // Update port number if changed
-    cache: new InMemoryCache(),
-  });
+import express from 'express';
+import { PrismaClient } from '@prisma/client';
+import env from 'dotenv';
+import bcrypt from 'bcrypt';
 
+const router = express.Router();
+const prisma = new PrismaClient();
+const saltRounds = 10;
+env.config();
 
-function App() {
-  const [user, setUser] = useState(() => {
-    const storedUser = localStorage.getItem('user');
+router.post('/signup', async (req, res) => {
```

Comment:
Signup looking a lot cleaner, thanks for adding proper error handling including status codes as well as salt for your password encryptio